feature/funding/googleplay/src/main/kotlin/app/k9mail/feature/funding/googleplay/ui/contribution/ContributionList.kt

```diff
@@ -260,3 +294,20 @@ private fun ListErrorView(
         )
     }
 }
+
+private fun Context.launchThunderbirdWebsite() {
```

Comment:
This is not necessary anymore, the url is handled by the system already.

Reply:
Fixed, Idk why but on my previous attempt it seems not working so I thought I had to handle through the listener. Now I tested again and it's opening the url properly.